test/Models/Experiment.ts

```diff
@@ -49,7 +49,7 @@ describe("NewStuff", function () {
       );
     });
 
-    autorun((dispose) => {
+    autorun((_dispose) => {
```

Comment:
Did you intend to remove the console.log in the next line too?

Reply:
This whole file isn't actually asserting anything and hasn't been touched since 2021, I'll check with the team about removing it completely.

Reply:
Deleted